md/response.md

```diff
@@ -0,0 +1,194 @@
+# Response
```

Comment:
Docs are ❤️ , but `@doc` attributes >>>> a random markdown file. 😉

Reply:
I agree! Added base `@doc` attributes as well.

Thank you!